src/imgui/ImGuiTools.hh

```diff
@@ -42,8 +42,8 @@ private:
 	std::string screenshotName;
 	enum class SsType : int { RENDERED, MSX, NUM };
 	int screenshotType = static_cast<int>(SsType::RENDERED);
-	enum class SsSize : int { S_320, S_640, NUM };
-	int screenshotSize = static_cast<int>(SsSize::S_320);
+	enum class SsSize : int { S_320, S_640, AUTO, NUM }; // keep order for bw compat of persisted element
```

Comment:
No need to change anything, just some background info.

Till now we never cared about bw compat for GUI settings.
And we know that we want to rework this in the future (e.g. split "recently used" from "GUI layout" etc).

Reply:
You're right about that. When we change it, we can remove that line :)